tests/View/ViewBladeCompilerTest.php

```diff
@@ -508,6 +508,14 @@ public function testUnsetStatementsAreCompiled()
         $this->assertEquals($expected, $compiler->compileString($string));
     }
 
+    public function testLiteralBlocksAreCompiled()
```

Comment:
It makes sense to test:
1) several lines in literal block
2) several literal blocks in a view